code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -76,7 +78,32 @@
 				else
 					C.deconstruct()
 					visible_message("<span class='warning'>[src] chews through the [C].</span>")
+					
+/mob/living/simple_animal/mouse/Move()
+	. = ..()
+	if(!stat)
+		eat_cheese()
 
+/mob/living/simple_animal/mouse/proc/eat_cheese()
+	var/obj/item/reagent_containers/food/snacks/cheesewedge/CW = locate(/obj/item/reagent_containers/food/snacks/cheesewedge) in loc
+	if(CW && full == FALSE)
+		qdel(CW)
```

Comment:
You are deleting the cheese here but later you reference it at line 92 in the message. Move this below that message